packages/dx-chart-core/src/types/chart-core.types.ts

```diff
@@ -102,6 +102,9 @@ export interface Series {
   readonly valueField: string;
   readonly createHitTester: CreateHitTesterFn;
   readonly getPointTransformer: GetPointTransformerFn;
+  readonly pointComponent: React.ComponentType<any>;
+  readonly state: string;
```

Comment:
By its meaning it should be `state?`. How does it work?

Reply:
The properties was added for avoid ts errors in [that](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-chart/src/utils/series-helper.jsx#L53) code. It is work because `seriesComponent` is any.